### PR TITLE
Add transformation-matrix-js w/ git auto-update via single package.json

### PR DESCRIPTION
**Pull request by extend1994**
Pull request for issue: #10763

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://gitlab.com/epistemex/transformation-matrix-js.git
 * NPM package url (optional): https://www.npmjs.com/package/transformation-matrix-js
 * License and its reference: [MIT](https://gitlab.com/epistemex/transformation-matrix-js#license)
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 0
   - Count of stars: 0
   - Count of forks: 0
 * NPM download stats (optional):
   - Downloads in the last day: 284
   - Downloads in the last week: 1,341
   - Downloads in the last month: 4,729

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [ ] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by extend1994** [comment]
@cdnjs/intern3 Please help me review this pull request, thank you.

File: ajax/libs/transformation-matrix-js/package.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "rotate",
+    "translate",
+    "scale",
+    "animate",
+    "animation",
+    "triangles"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/epistemex/transformation-matrix-js.git"
```
Comment (hare1039) [inline ajax/libs/transformation-matrix-js/package.json]: This url returns 404 page not found.
Need to change to gitlab's url

File: ajax/libs/transformation-matrix-js/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  },
+  "license": "MIT",
+  "npmName": "transformation-matrix-js",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "matrix.js"
```
Comment (hare1039) [inline ajax/libs/transformation-matrix-js/package.json]: I recommend to use `matrix*`
What do you think?

**Comment by extend1994** [comment]
@hare1039 I modified the git repo url and auto-update config. Please help me review it again, thanks.

ping @sufuf3 @clairetsai818

**Comment by pvnr0082t** [comment]
It would be good to use gitlab.

**Comment by extend1994** [comment]
@sufuf3 @pvnr0082t I modified the auto-update config, please help me review it, thank you.

**Comment by extend1994** [comment]
@PeterDaveHello Please help me review this pull request, thank you.

**Comment by PeterDaveHello** [comment]
Guys please don't write the name wrong - `gitlab` -> `GitLab`, I remember that we have deal about being polite not being rude.

**Comment by PeterDaveHello** [comment]
@pvnr0082t  @cdnjs/intern3 please note that GitLab doesn't support `git` protocol actually, I'm not sure if you noticed it or not, the autoupdater actually will not work properly on it. Anyway I'll add https support soon.

**Comment by PeterDaveHello** [comment]
Looks like the repo is dead now ... @extend1994 would you please help take a look?

**Comment by extend1994** [comment]
@PeterDaveHello I checked it's dead. The author's account even doesn't exist on GitLab now. The author's GitHub account exists but there is no available email or contact information. The only possible contact information is github@epistemex.com which is in `package.json`. I am not sure what can I do next. Try to contact the author by e-mail or tag the author...?

**Comment by PeterDaveHello** [comment]
Since there're many works need to be done first, when you are really free, can try to contact the author.

**Comment by extend1994** [comment]
@PeterDaveHello Okay!

**Pull request merged**